contracts/interfaces/ICallbackGnosis.sol

```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+import "./IProxyCreationCallback.sol";
+
+
+interface ICallbackGnosis is IProxyCreationCallback {
+    /// @dev Method called once a proxy contract is created
+    /// @param proxy GnosisSafe address
+    /// @param _singleton GnosisSafe impl address
+    /// @param initializer Payload used to setup GnosisSafe Configuration
+    /// @param saltNonce Salt utilized for GnosisSafe Create2 opcode
+    function proxyCreated(
+        address proxy,
+        address _singleton,
+        bytes calldata initializer,
+        uint256 saltNonce
+    ) external;
```

Comment:
Since this interface _is_ IProxyCreationCallback, and proxyCreated is defined in IProxyCreationCallback, do we need to declare it here as well?